pkg/env/env.go

```diff
@@ -0,0 +1,167 @@
+package env
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+)
+
+// Bind updates the Go variable via the pointer with the value of the
+// environment variable, if set and not empty.
+//
+// An error is returned if it failed to get the environment variable.
+func Bind(ptr *string, key string) error {
+	if value, ok := os.LookupEnv(key); ok {
+		*ptr = value
+	}
+	return nil // can't fail, but just for consitency with the other funcs
+}
+
+// BindMultiple updates the Go variables via the pointers with the values of the
+// environment variables, if set and not empty, for each respective pair in
+// the map.
+//
+// An error is returned if it failed to get any of the environment variables for
+// the mappings.
```

Comment:
Should maybe change `mappings` to `bindings` to stay consistent with the function names?

Reply:
Good idea!